pkgs/by-name/pr/proton-ge-rtsp-bin/package.nix

```diff
@@ -9,13 +9,19 @@
 proton-ge-bin.overrideAttrs (
   finalAttrs: _: {
     pname = "proton-ge-rtsp-bin";
-    version = "GE-Proton9-20-rtsp16";
+    version = "GE-Proton9-22-rtsp17";
 
     src = fetchzip {
       url = "https://github.com/SpookySkeletons/proton-ge-rtsp/releases/download/${finalAttrs.version}/${finalAttrs.version}.tar.gz";
       hash = "sha256-iq7oiDW5+51wzqYwASOGSV922c/pg1k29MdkIXlT34k=";
```

Comment:
This hash needs to be updated because of the new version

Reply:
done